app/assets/javascripts/components/projects/ProjectBoard.js

```diff
@@ -21,134 +28,150 @@ import SearchResults from './../search/SearchResults';
 import ProjectLoading from './ProjectLoading';
 import SideBar from './SideBar';
 import Columns from '../Columns';
-import Stories from "../stories/Stories";
-import Sprints from "../stories/Sprints";
+import { DragDropContext } from 'react-beautiful-dnd';
 
 export const ProjectBoard = ({
   fetchProjectBoard,
   projectId,
   projectBoard,
-  createStory,
   closeHistory,
   notifications,
   removeNotification,
   history,
   chillyBinStories,
   backlogSprints,
-  doneSprints,
-  fetchPastStories,
   toggleColumn,
-  reverseColumns
+  reverseColumns,
+  doneSprints,
+  createStory,
+  dragDropStory,
+  fetchPastStories
 }) => {
+  const [newChillyBinStories, setNewChillyBinStories] = useState([]);
+  const [newBacklogSprints, setNewBacklogSprints] = useState([]);
+
+  useEffect(() => {
+    setNewBacklogSprints(backlogSprints);
+  }, [backlogSprints]);
```

Comment:
Since you are using `setNewBacklogSprints` function, you should include it into dependencies array.
```suggestion
  }, [setNewBacklogSprints , backlogSprints]);
```

https://stackoverflow.com/a/57180908